cli/azd/cmd/auth_login.go

```diff
@@ -254,81 +257,103 @@ func (la *loginAction) Run(ctx context.Context) (*actions.ActionResult, error) {
 			"Next time use `azd auth login`.")
 	}
 
-	if !la.flags.onlyCheckStatus {
-		if err := la.accountSubManager.ClearSubscriptions(ctx); err != nil {
-			log.Printf("failed clearing subscriptions: %v", err)
+	if la.flags.onlyCheckStatus {
+		// In check status mode, we always print the final status to stdout.
+		// We print any non-setup related errors to stderr.
```

Comment:
@ellismg Slight behavioral change here to print unexpected errors to `stderr`. Is this okay for our editor integrations?

Reply:
I just looked and this should be fine - they ignore stderr on these paths today.